src/main/java/CinemaRoomRestService/RestService/api/dto/CinemaRoomDTO.java

```diff
@@ -0,0 +1,13 @@
+package CinemaRoomRestService.RestService.api.dto;
+
+import CinemaRoomRestService.RestService.domain.entity.Seat;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+
+@Builder
+@Getter
+public class CinemaRoomDTO {
+    private List<Seat> allSeats;
```

Comment:
it is a bad practice to Store Domain object inside DTO object, You should use SeatDTO here